.vscode/settings.json

```diff
@@ -1,6 +1,6 @@
 {
   "editor.codeActionsOnSave": {
-    "source.fixAll.eslint": true
+    "source.fixAll.eslint": "explicit"
```

Comment:
What does this do?

Reply:
I have no idea.  It just keeps getting auto updated in my local vscode settings.  Lemme remove it.  Didn't mean to commit but it just keeps showing up.